frontend/src/App.tsx

```diff
@@ -7,7 +7,7 @@ import MissionInformation from './components/Overlay/MissionInformation';
 import OverlayWelcome from './components/Overlay/OverlayWelcome';
 import ResetProgressOverlay from './components/Overlay/ResetProgress';
 import { LEVEL_NAMES } from './models/level';
-import { levelService } from './service';
+import { resetService } from './service';
```

Comment:
Yep that's fine, and I assume when you get around to tackling the reset level call, it will go in resetService as well?

Reply:
that's that plan!